integrationTests/mock/statusMetricsStub.go

```diff
@@ -2,54 +2,54 @@ package mock
 
 // StatusMetricsStub -
 type StatusMetricsStub struct {
```

Comment:
i think we can remove also this mock and reference the newly created one from testscommon

Reply:
good catch. done